py3status/modules/systemd_suspend_inhibitor.py

```diff
@@ -0,0 +1,100 @@
+# -*- coding: utf-8 -*-
+"""
+Turn on and off systemd suspend inhibitor.
+
+Configuration parameters:
+    format: display format for this module
+        (default '[\?color=state SUSPEND [\?if=state OFF|ON]]')
+    lock_types: specify state to inhibit, comma separated list
+        https://www.freedesktop.org/wiki/Software/systemd/inhibit/
+        (default 'handle-lid-switch,idle,sleep')
+    thresholds: specify color thresholds to use
+        (default [(True, 'bad'), (False, 'good')])
+
+Format placeholders:
+    {state} systemd suspend inhibitor state, eg True, False
+
+Color thresholds:
+    xxx: print a color based on the value of `xxx` placeholder
+```
+
+
+Examples:
```

Comment:
Examples is in wrong place (should be outside of ``` above). Alternatively, kill this since it is same as default.

Reply:
Examples removed.